src/annotations/AnnotationParser.php

```diff
@@ -122,7 +122,8 @@ public function parse($source, $path)
                         $use .= $str;
                     } elseif ($type === self::CHAR) {
                         if ($str === ',' || $str === ';') {
-                            $uses[substr($use, 1 + strrpos($use, '\\'))] = $use;
+                            $use_components = explode('\\', $use);
```

Comment:
This line uses the `explode` function side effect that if char for explode wasn't found in a string the whole string is returned as 1st array element.

It's better to be more specific (this code will replace this and following line):
```php
if (strpos($use, '\\') !== false) {
    $uses[substr($use, 1 + strrpos($use, '\\'))] = $use;
}
else {
    $uses[$use] = $use;
}
```